src/common/cryptoutils.cpp

```diff
@@ -19,6 +19,59 @@ constexpr auto cSchemeFile      = "file";
 constexpr auto cSchemePKCS11    = "pkcs11";
 constexpr auto cSchemeMaxLength = Max(sizeof(cSchemeFile), sizeof(cSchemePKCS11));
 
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+static Error StrToPKCS11ID(const String& idStr, Array<uint8_t>& id)
```

Comment:
Not a good name: I would suggest DecodePKCS11ID

Reply:
done